integration/client-cli/admin-cli/src/main/java/org/keycloak/client/cli/common/BaseConfigCredentialsCmd.java

```diff
@@ -115,6 +115,7 @@ public void process() {
         } else if (keystore != null || secret != null || clientSet) {
             grantTypeForAuthentication = OAuth2Constants.CLIENT_CREDENTIALS;
             printErr("Logging into " + server + " as " + "service-account-" + clientId + " of realm " + realm);
+            printErr("WARN service-account-" + clientId + " is a temporary account. To harden security, create a permanent account and delete the temporary one.");
```

Comment:
As discussed offline, this warning rather belongs to the server and not (only) to the admin client. Also, the warning should consider both user and service account.